ludwig/utils/audio_utils.py

```diff
@@ -0,0 +1,99 @@
+#! /usr/bin/env python
+# coding=utf-8
+# Copyright (c) 2019 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import numpy as np
+from scipy.signal.windows import get_window
+from scipy.signal import lfilter
+
+def _pre_emphasize_data(data, emphasize_value=0.97):
+    filter_window = np.asarray([1, -emphasize_value])
+    pre_emphasized_data = lfilter(filter_window, 1, data)
+    return pre_emphasized_data
+
+def get_length_in_samp(sampling_rate_in_hz, length_in_s):
+    return int(sampling_rate_in_hz * length_in_s)
+
+def get_group_delay(raw_data, sampling_rate_in_hz, window_length_in_s, window_shift_in_s, num_fft_points, window_type):
```

Comment:
@patrickvonplaten Could you please add brief descriptions for these methods? like docstrings